src/Main.ts

```diff
@@ -886,7 +886,7 @@ export class Main {
                 teamId = await this.clientFactory.upsertTeamByToken(opts.slack_bot_token);
                 log.info(`Found ${teamId} for token`);
             } catch (ex) {
-                log.error("Failed to action link because the token couldn't used:", ex);
+                log.error("Failed to action link because the token couldn't be used:", ex);
```

Comment:
I'll let you off, but this should have been a seperate PR because it's not really relevant to the context :p